tests/ts_calibrate.c

```diff
@@ -182,6 +182,12 @@ static int ts_validate(struct tsdev *ts, int boundary, unsigned int loops)
 	fillrect(0, 0, xres - 1, yres - 1, 0);
 	put_string_center(xres / 2, yres / 4, textbuf, 1);
 
+	if(timeout == 0){
```

Comment:
please fix the coding style

Reply:
Please recheck.

Reply:
still, a space between if and the open bracket seems to be missing...

also, could you squash your 3 changes to be only one commit? It all is one change, adding the -s option that closes the framebuffer.

other than that, I like it. Thanks a lot!

Reply:
sorry I missed that completely ;)